From: Alexander Dahl <ada@thorsis.com>
Date: Fri, 23 Feb 2018 08:50:04 +0100
Subject: [PATCH] Remove -Werror

Otherwise build fails on arm-v5te if global -Wcast-align is set.

Signed-off-by: Alexander Dahl <ada@thorsis.com>
---
 config/eu.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/config/eu.am b/config/eu.am
index 8fe1e259f9e2..c5a6209a4e04 100644
--- a/config/eu.am
+++ b/config/eu.am
@@ -71,7 +71,6 @@ AM_CFLAGS = -std=gnu99 -Wall -Wshadow -Wformat=2 \
 	    -Wold-style-definition -Wstrict-prototypes \
 	    $(LOGICAL_OP_WARNING) $(DUPLICATED_COND_WARNING) \
 	    $(NULL_DEREFERENCE_WARNING) $(IMPLICIT_FALLTHROUGH_WARNING) \
-	    $(if $($(*F)_no_Werror),,-Werror) \
 	    $(if $($(*F)_no_Wunused),,-Wunused -Wextra) \
 	    $(if $($(*F)_no_Wstack_usage),,$(STACK_USAGE_WARNING)) \
 	    $($(*F)_CFLAGS)
